testing/munin/munin-node.post-install

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# create directories
+mkdir -p /usr/share/munin/docs
+mkdir -p /var/lib/munin/spool
+mkdir -p /var/log/munin
```

Comment:
`/var/lib/munin/spool` and `/var/log/munin` is IMO better to create in runscript using `checkpath`. Then you can completely omit these `post-{install,upgrade}` scripts.